frontend/src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import Button from './Button'
+import ModalLink from './ModalLink'
+
+const OverlayContainer = styled.div<{ overlayBackgroundColor: string }>`
+  position: fixed;
+  min-height: 100vh;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: ${(props) => (props.overlayBackgroundColor ? props.overlayBackgroundColor : 'rgba(0, 0, 0, 0.3)')};
+  z-index: 999;
+`
+
+const MainContainer = styled.div`
+  position: fixed;
+  background-color: #fff;
+  width: 90vw;
+  max-width: 26em;
+  border-radius: 15px;
+  margin: auto;
+  padding: 15px;
+  box-shadow: 0 2px 5px 1px #888888;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+`
+
+const TitleText = styled.h1`
+  font-family: Lato;
+  font-size: 22px;
+  text-align: center;
+`
+
+const DescriptionText = styled.p`
+  font-family: Lato;
+  font-size: 14px;
+  text-align: center;
+  position: relative;
+  top: -10px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+  margin-top: 10px;
+  position: relative;
+  top: -10px;
+`
+
+type Props = {
+  link?: string
+  onLinkClick: React.MouseEventHandler<HTMLElement>
+  title: string
+  description?: string | JSX.Element
+  // Left Button Props
+  hasLeftButton?: boolean
+  leftButtonType: 'secondary'
```

Comment:
not sure if this is necessary? if it can only be of 1 type, would it be better to not require leftButtonType at all?

Reply:
Resolved in commit 8a39ccd6e0e18c9e6e1d9c931c1355c8e549ffdb.